packages/table/assets/index.css

```
@@ -74,3 +74,61 @@
+    --vc-table-stack-breakpoint: 640px;
+}
+
+@media (max-width: 640px) {
+    .vc-table[data-responsive="true"] thead {
+        position: absolute;
+        width: 1px;
+        height: 1px;
+        margin: -1px;
+        padding: 0;
+        overflow: hidden;
+        clip: rect(0 0 0 0);
```

Stylelint (17.11.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)